services-gateway-http/src/main/java/io/scalecube/services/gateway/http/HttpGatewayAcceptor.java

```diff
@@ -64,6 +64,8 @@
     return httpRequest
         .receive()
         .aggregate()
+        .switchIfEmpty(
+            Mono.defer(() -> ByteBufMono.just(ByteBufAllocator.DEFAULT.compositeBuffer())))
```

Comment:
what is composite buffer maybe we need an empty buffer?

Reply:
Changed